smartsim/_core/_install/buildenv.py

```diff
@@ -211,10 +211,11 @@ class Versioner:
     ONNX = Version_(REDISAI.onnx)
 
     def as_dict(self):
+        database_name = "REDIS" if "redis" in self.REDIS_URL else "KEYDB"
```

Comment:
What if a user installed redis from a different url than the standard? their renamed fork? unlikely but possible.